src/component/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { NavLink, withRouter } from "react-router-dom";
+
+/**
+ * Displays information about the app and a menu of choices.
+ * @returns {JSX.Element}
+ */
+function Header (props) {
+    const { pathname } = props.location;
+
+    return (
+        <header>
+            <nav>
+                <h1>place-my-order.com</h1>
+                <ul>
+                    <li className={pathname === "/" ? "active" : ""}><NavLink to="/">Home</NavLink></li>
```

Comment:
Does `NavLink` do anything different than `Link` when you aren't giving it activeClass? I thought that's really all it did.

Reply:
Good point, changed to Link.